### DISCUSSION TITLE
Players skins

### DISCUSSION THREAD

**Discussion by Complas** [Sam54123/mc-world-export#27] [Q&A] [answered]
Is there a way to get the players skins from the replay on blender?

**Comment by Igrium** [comment, marked as answer]
That used to happen automatically. Strange. In any case, it should be as simple as replacing the player texture in Blender.

**Reply by Complas** [reply]
maybe an option that i do not activate? manualy changing texture for each player is too much

**Reply by Complas** [reply]
and the block are invisible

https://user-images.githubusercontent.com/102386071/213882798-f9503df9-259a-45c5-b262-1d5abda99acf.mp4

**Reply by Igrium** [reply]
There isn't an option because it supposed to use the correct skin no matter what. This is a bug; I'll write it down.

**Reply by Igrium** [reply]
The blocks being invisible is a known issue. Just save the blend file and re-open it, and it should be fixed. See #21

**Reply by Complas** [reply]
ok thanks
